packages/cli/src/bin/ceramic.ts

```diff
@@ -63,8 +65,13 @@ program
             logDirectory,
             network,
             pubsubTopic,
-            corsAllowedOrigins
-        )
+            corsAllowedOrigins,
+            disableAnchors,
+        ).catch((err) => {
```

Comment:
without this starting up without an anchor service and without specifying `--disable-anchors` causes the process to hang at startup, rather than fully fail and exit

Reply:
+1